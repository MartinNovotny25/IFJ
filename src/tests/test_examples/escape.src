print('\n')
print('\f')
print('\a')
print('\b')
print('\r')
print('\t')
print('\v')
print('\\')
print('\'')
print('\"')
print('"')
print('\?')
print('\x61')
print('\0')
""" test \n test """
""" test
\n
test """
""" test \f """
""" test \"" """
""" test "\" test """
""" test '\n' test """
""" test ' ' test """
""" test "\\" test """
""" test
test """
a = 'Ahoj\\\'\Ahoj'
print(a)
print('\x1')
