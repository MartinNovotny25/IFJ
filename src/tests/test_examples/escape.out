1 -  chyba v programu v ramci lexikalni analyzy.

TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, 
)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, \f)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, \a)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, \b)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, \r)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, 	)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, \v)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, \)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, ')
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, ")
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, ")
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, \?)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, a)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, \0)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, Ahoj\'\Ahoj)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
